macros/B2GOnlyHeader2.ejs

```diff
@@ -37,6 +43,8 @@ var secString = {
     privileged: mdn.localString({
         "en-US" : 'This API is available on <a href="/en-US/docs/Mozilla/Firefox_OS">Firefox OS</a> for ' +
                   '<a href="/en-US/docs/Mozilla/Firefox_OS/Security/Application_security#App_Types">privileged or certified applications</a> only.',
+        "zh-CN" : '此 API 仅在 <a href="/zh-CN/docs/Mozilla/Firefox_OS">Firefox OS</a> 的' +
+                  '<a href="/zh-CN/docs/Mozilla/Firefox_OS/Security/Application_security#App_Types">高权限或已认证应用程序</a>上可用。',
```

Comment:
👍 tested on a translation of [Archive/B2G_OS/API/UDPSocket](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/API/UDPSocket)